### PR TITLE
Clarify instructions to install operator via OLM

### PR DESCRIPTION
**Pull request by kylape**
I had trouble deploying the operator via OLM using the given instructions, so I've updated them here in an attempt to fill in the gaps where I got stuck along the way.

Looks like the diff is way more confusing than I feel like it needs to be, sorry.

Changes:

* Renamed "Launch the Operator with OLM and Index" to "Installing operator via OLM"
* Added a prerequisites section
* Explained how to run the `olm-operator-install.sh` script if desired

## Testing Performed

I developed the updated instructions while working with the commands locally.  I suggest reviewers do the same if they want to verify correctness!

### CODE REVIEW

File: operator/README.md [hunk 1]
```diff
@@ -231,46 +231,82 @@ $ make olm-uninstall
...
-make kuttl deploy-dirty-tag-via-olm
+ROX_PRODUCT_BRANDING=RHACS_BRANDING make deploy-via-olm TEST_NAMESPACE=my-favorite-namespace
 ```
 
-For upgrading an existing operator:
+If you'd rather put in a custom image spec, you can use the install script directly:
 
 ```bash
-make kuttl upgrade-via-olm
+hack/olm-operator-install.sh stackrox-operator quay.io/rhacs-eng/stackrox-operator 3.74.0-588-ge99fe7b316-dirty
```
Comment (theencee) [inline operator/README.md:299]: Remove `-dirty` from tag? Especially given the warning on line 274

File: operator/README.md [hunk 2]
```diff
@@ -231,46 +231,82 @@ $ make olm-uninstall
...
+If you'd rather put in a custom image spec, you can use the install script directly:
 
 ```bash
-make kuttl upgrade-via-olm
+hack/olm-operator-install.sh stackrox-operator quay.io/rhacs-eng/stackrox-operator 3.74.0-588-ge99fe7b316
+```
+
+### Removal
 
+You can blow everything away with:
```
Comment (johannes94) [inline operator/README.md:304]: Nit: I'd prefer different wording:
```suggestion
You can tear everything down with:
```

File: operator/README.md [hunk 3]
```diff
@@ -231,46 +231,82 @@ $ make olm-uninstall
...
+
+```bash
+$ make olm-uninstall
+$ kubectl delete ns stackrox-operator
```
Comment (johannes94) [inline operator/README.md:308]: Consider also running `make uninstall` to remove the CRDs from the cluster.

**Review by johannes94 — CHANGES_REQUESTED** [review]

File: operator/README.md (hunk 3 above)
Comment (kylape) [reply operator/README.md:308]: Thanks, added.

**Review by johannes94 — APPROVED** [review]

**Pull request merged**